### PR TITLE
Finish Static Pages

### PR DESCRIPTION
**Pull request by vjnhat**
change update Gemfile and modify database.yml file use sqlite3 instead mysql2

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# SQLite version 3.x
```
Comment (dieunb) [inline config/database.yml]: Checkout this file, don't push into repo

File: config/database.yml.example [hunk 2]
```diff
@@ -1,54 +0,0 @@
-# MySQL.  Versions 5.0+ are recommended.
```
Comment (dieunb) [inline config/database.yml.example]: don't remove this file. Pls checkout it

**Comment by leduytungframgia** [comment]
nhiều hơn 10 file.

File: app/assets/javascripts/static_pages.coffee [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (dieunb) [inline app/assets/javascripts/static_pages.coffee]: remove this file

File: app/assets/stylesheets/static_pages.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the static_pages controller here.
```
Comment (dieunb) [inline app/assets/stylesheets/static_pages.scss]: remove this file

File: app/helpers/application_helper.rb [hunk 5]
```diff
@@ -1,2 +1,13 @@
 module ApplicationHelper
+
+
+# Returns the full title on a per-page basis.
```
Comment (dieunb) [inline app/helpers/application_helper.rb]: remove all remark

File: app/helpers/static_pages_helper.rb [hunk 6]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
```
Comment (dieunb) [inline app/helpers/static_pages_helper.rb]: remove this file

File: config/environments/production.rb [hunk 7]
```diff
@@ -76,4 +76,7 @@
 
   # Do not dump schema after migrations.
```
Comment (leduytungframgia) [inline config/environments/production.rb]: remove remark

**Comment by leduytungframgia** [comment]
OK

File: app/helpers/application_helper.rb [hunk 8]
```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+
+  def full_title(page_title = '')
```
Comment (dieunb) [inline app/helpers/application_helper.rb]: remove `()`

File: app/views/layouts/application.html.erb [hunk 9]
```diff
@@ -1,14 +1,18 @@
...
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
-  <%= csrf_meta_tags %>
-</head>
-<body>
-
-<%= yield %>
-
-</body>
+  <head>
+    <title><%= full_title(yield(:title)) %></title>
```
Comment (dieunb) [inline app/views/layouts/application.html.erb]:
```
 full_title yield(:title)
```

**Pull request merged**